superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -787,7 +787,11 @@ export function queryEditorSetSchema(queryEditor, schema) {
 
     return sync
       .then(() =>
-        dispatch({ type: QUERY_EDITOR_SET_SCHEMA, queryEditor, schema }),
+        dispatch({
+          type: QUERY_EDITOR_SET_SCHEMA,
+          queryEditor: queryEditor || {},
+          schema: schema || {},
```

Comment:
@pkdotson or @hughhhh this seems wrong, i.e., schema is a nullable string and not an object. Should this be `schema: schema || null`?

Reply:
@john-bodley you're right - here's the fix: #19664 (caught during testing of 1.5)